config/releases.exs

```diff
@@ -1,16 +1,18 @@
 import Config
 
-producer_config =
-  "SQS_PRODUCER"
+config =
+  "CONFIG"
   |> System.fetch_env!()
   |> Jason.decode!()
 
 config :copy_cat,
   producer:
     {BroadwaySQS.Producer,
-     queue_url: producer_config["queue_url"],
+     queue_url: config["COPYCAT_SQS_URL"],
```

Comment:
@btkostner should we keep this ENV name specific to the app or standardize on `SQL_URL` instead?

Reply:
ummm, I think we are going to just have a single queue url for each project. So probably `SQL_URL`. Keeps it easier when we create new projects